src/pages/NewSensor/BreachConfigRow.js

```diff
@@ -16,58 +15,34 @@ export const TYPE_TO_LABEL = {
   COLD_CUMULATIVE: 'Cold cumulative',
 };
 
-export const BreachConfigRowComponent = ({
-  type,
-  duration,
-  temperature,
-  updateDuration,
-  updateTemperature,
-  threshold,
-}) => {
-  const isHotBreach = type.includes('HOT');
-  const Icon = isHotBreach ? HotBreachIcon : ColdBreachIcon;
-  const color = isHotBreach ? DANGER_RED : COLD_BREACH_BLUE;
-
-  return (
-    <EditorRow Icon={<Icon color={color} size={20} />} label={TYPE_TO_LABEL[type]}>
-      <DurationEditor value={duration / SECONDS.ONE_MINUTE} onChange={updateDuration} />
-      <TemperatureEditor
-        above={isHotBreach}
-        value={temperature}
-        onChange={updateTemperature}
-        threshold={threshold}
-      />
-    </EditorRow>
-  );
-};
-const stateToProps = (state, props) => {
-  const { type } = props;
-  const { newSensor } = state;
-  const config = newSensor[type];
-  const { duration, temperature } = config;
-
-  const thresholds = selectConfigTemperatureThresholds(state);
-
-  return { duration, temperature, threshold: thresholds[type] };
-};
-
-const dispatchToProps = (dispatch, props) => {
-  const { type } = props;
-
-  const updateDuration = value => dispatch(NewSensorActions.updateConfig(type, 'duration', value));
-  const updateTemperature = value =>
-    dispatch(NewSensorActions.updateConfig(type, 'temperature', value));
-
-  return { updateDuration, updateTemperature };
-};
-
-BreachConfigRowComponent.propTypes = {
+export const BreachConfigRow = React.memo(
```

Comment:
Changed this component and disconnected it from redux as I went with different actions for editing a sensor and creating a new one.